cmd/paramgen/internal/paramgen.go

```diff
@@ -123,6 +123,18 @@ func parsePackage(path string) (*ast.Package, error) {
 	if len(pkgs) == 0 {
 		return nil, fmt.Errorf("no source-code package in directory %s", path)
 	}
+	// Ignore files with go:build constraint set to "tools" (common pattern in
+	// Conduit connectors).
+	for pkgName, pkg := range pkgs {
+		for fileName, f := range pkg.Files {
+			if hasBuildConstraint(f, "tools") {
+				delete(pkg.Files, fileName)
+			}
+		}
```

Comment:
Alternatively, `maps.DeleteFunc(pkg.Files, hasBuildConstaint)`.

Reply:
Nice, I'll change it 👍